File: openapi-generator/dart/lib/model/inline_object4.dart
part of openapi.api;

class InlineObject4 {
  /* True indicates that this driver should be reactivated. */
  bool reactivate = null;
  InlineObject4();

  @override
  String toString() {
    return 'InlineObject4[reactivate=$reactivate, ]';
  }

  InlineObject4.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    if (json['reactivate'] == null) {
      reactivate = null;
    } else {
          reactivate = json['reactivate'];
    }
  }

  Map<String, dynamic> toJson() {
    return {
      'reactivate': reactivate
    };
  }

  static List<InlineObject4> listFromJson(List<dynamic> json) {
    return json == null ? new List<InlineObject4>() : json.map((value) => new InlineObject4.fromJson(value)).toList();
  }

  static Map<String, InlineObject4> mapFromJson(Map<String, dynamic> json) {
    var map = new Map<String, InlineObject4>();
    if (json != null && json.length > 0) {
      json.forEach((String key, dynamic value) => map[key] = new InlineObject4.fromJson(value));
    }
    return map;
  }
}

File: openapi-generator/csharp-dotnet2/src/main/CsharpDotNet2/Org/OpenAPITools/Model/InlineObject4.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace Org.OpenAPITools.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class InlineObject4 {
    /// <summary>
    /// True indicates that this driver should be reactivated.
    /// </summary>
    /// <value>True indicates that this driver should be reactivated.</value>
    [DataMember(Name="reactivate", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "reactivate")]
    public bool? Reactivate { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class InlineObject4 {\n");
      sb.Append("  Reactivate: ").Append(Reactivate).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}
